src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+import java.util.Scanner;
+
+/**
+ * Chatbot program Duke that tracks user tasks.
+ *
+ * @author Leow Yuan Yang
+ * @version 4.0
+ * @since 2021-08-25
+ */
 public class Duke {
+    private static final String SEPARATOR = "\t==============================================";
+    private static final String logoArt = "\t                                        \n" +
+            "\t             ..........                 \n" +
+            "\t           .,,..,.,,,,                  \n" +
+            "\t           ,.,,,,.,...                  \n" +
+            "\t          ,.......,*,/& .               \n" +
+            "\t          ,..,..,((/**(((,              \n" +
+            "\t           *(#%(((#/#((/       *        \n" +
+            "\t         /(%#########%##(*,,,((*        \n" +
+            "\t              ((#(((((#      (((*       \n" +
+            "\t        ((((((   /((((/,., .... #       \n" +
+            "\t         (((#    .(((..*...,*///        \n" +
+            "\t         ((/      .,,**  ..**///        \n" +
+            "\t                  .,*,..  ,**//         \n" +
+            "\t                ..,,,..,/ ,*//          \n" +
+            "\t             *,,****,        " ;
+    private static final String nameArt = "\t _______  __   __  __   __  ______    _______ \n" +
+            "\t|       ||  |_|  ||  | |  ||    _ |  |       |\n" +
+            "\t|  _____||       ||  | |  ||   | ||  |    ___|\n" +
+            "\t| |_____ |       ||  |_|  ||   |_||_ |   |___ \n" +
+            "\t|_____  ||       ||       ||    __  ||    ___|\n" +
+            "\t _____| || ||_|| ||       ||   |  | ||   |    \n" +
+            "\t|_______||_|   |_||_______||___|  |_||___|    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        boolean isDone = false;
+        Task[] taskList = new Task[100];
+        Scanner in = new Scanner(System.in);
+
+        printWelcomeMessage();
+
+        do {
+            // Read inputs and split them into substrings
+            String userInput = readUserInput(in);
+            String[] userInputSubstrings = userInput.strip().split(" ", 2);
+            String command = userInputSubstrings[0];
+
+            switch (command) {
+            case "bye":
+                isDone = true;
+                printExitMessage();
+                break;
+            case "done":
+                try {
+                    int itemNum = Integer.parseInt(userInput.replaceAll("[^0-9]", ""));
+                    taskList[itemNum - 1].setDone();
+                    printDoneTask(taskList[itemNum - 1], itemNum);
```

Comment:
Maybe instead of itemNum, you could use taskNum, since we are talking about tasks.